### ISSUE TITLE
Add toggle feature to Power On/Off

### ISSUE THREAD

**Issue by BenJamesAndo** [bitfocus/companion-module-epson-businesspj#3]
I saw PJLink module just got this [functionality](https://github.com/bitfocus/companion-module-generic-pjlink/releases/tag/v1.2.4). It would be nice to have it here as well, if possible. I specifically like how when you press power off it shows the power menu on the projector and that you have to press it twice. So I suppose the toggle would work that if the projector is off pressing it once would turn it on. Then pressing it twice to turn it off via toggle.

**Comment by h07d0q** [comment]
I added an PR #16 which provides Power on/off with only one button. short press sends "on" command and longpress sends Power off command immediately twice but you can edit the action easily to send the off command only once, so you still have to press the button long twice.
You can download my built module here:
[epson-businesspj.zip](https://github.com/user-attachments/files/18634408/epson-businesspj.zip)
